upgrade_tests/helpers/existence.py

```diff
@@ -72,11 +73,11 @@
         'medium',
         'organization',
         'os',
-        'partition-table',
         'puppet-class',
         'puppet-module',
         'repository',
         'role',
+        'sc-param',
```

Comment:
This was needed to move to dict of entities with unique id as name. But I moved with wrong observation of failure in PR #21 .